src/main/java/klerer/fallingsand/Sand.java

```diff
@@ -0,0 +1,50 @@
+package klerer.fallingsand;
+
+public class Sand {
+
+    private int[][] field = new int[3][3];
+    private int x;
+    private int y;
+
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
+        for (int y = 0; y < 3; y++) {
+            for (int x = 0; x < 3; x++) {
+                builder.append(field[y][x]);
+            }
+            builder.append("\n");
+        }
+        return builder.toString();
+    }
+
+    /**
+     * @return the value in field
+     */
+    public int get(int x, int y) {
+        return field[y][x];
+    }
+
+    /**
+     * sets the value in field to 1
+     */
+    public void put(int x, int y) {
+        field[y][x] = 1;
+    }
+
+    public void fall() {
+        // moves all sand down one square
+        if (get(x, y) == field[1][2]) {
```

Comment:
What are the values of x and y?

Reply:
the values in the array

Reply:
No. That's not the way variables work...

Reply:
do i need to pass them in as parameters or I'm supposed to do it a different way?

Reply:
The comment says, "move the sand down one square"
The fall() method takes no arguments.

You are supposed to have an idea how to do that.